cmd/promu.go

```diff
@@ -156,3 +156,12 @@ func envOr(name, def string) string {
 	}
 	return s
 }
+
+func checkRequiredConfigurations(configVars []string) {
+	for _, configVar := range configVars {
+		if !viper.IsSet(configVar) {
+			err := fmt.Errorf("'%s'", configVar)
+			fatalMsg(err, "Missing required configuration")
```

Comment:
The standard logging and fmt functions in go all have the interface `func(msg string, err error)`. Any reason to switch the arguments here?